README.md

```diff
@@ -1,18 +1,14 @@
 # Security Special Interest Group
 
-Covers horizontal security initiatives for the OpenTelemetry project, including
-regular security audits, the vulnerability management process, cross-cutting
-security documentation, and security community management.
+Covers horizontal security initiatives for the OpenTelemetry project, including regular security audits, the
+vulnerability management process, cross-cutting security documentation, and security community management.
 
-The [charter](./sig-security-charter.md) draft defines the scope and governance
-of the Security Special Interest Group.
+The [charter](./sig-security-charter.md) draft defines the scope and governance of the Security Special Interest Group.
 
 ## Meetings
 
-* Regular SIG Meeting: Bi-weekly on Wednesdays at [14:00
-  UTC](https://dateful.com/convert/utc?t=14).
-  * [Meeting notes and
-    Agenda](https://docs.google.com/document/d/1P2xejC7lEkOV_Z-8E0oZPXLK5HOnUPNuRqKP0ZQ5fpg).
+* Regular SIG Meeting: Bi-weekly on Wednesdays at [14:00 UTC](https://dateful.com/convert/utc?t=14).
```

Comment:
Was it not 12:00 UTC?

Reply:
We changed it before merging, so that it would be during a reasonable time for @codeboten.

Reply:
thanks for the clarification